composer.lock

```diff
@@ -4,7 +4,7 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "e86af25d9a4a1942c437cca58f9f1efb",
+    "content-hash": "f3674961f96b48fdd025a6c94610c8eb",
```

Comment:
We shouldn't need to update composer.lock for this

Reply:
yes, but without it, my local test fails

Reply:
Ok, so actually this change matches the hash in mainline, so I don't know why it is showing up as a change here. Maybe just need to sync up your branch. But it should be fine.